package.json

```diff
@@ -54,10 +54,12 @@
     "site": "bundle exec jekyll serve",
 
     "lint": "tslint --project tsconfig.json -c tslint.json --type-check",
-    "test": "npm run tsc && npm run test:only && npm run lint",
+    "test": "npm run build:only && npm run test:only && npm run test:runtime && npm run lint",
     "posttest": "npm run schema && npm run data && npm run mocha:examples",
-    "test:nocompile": "npm run test:only && npm run lint && npm run mocha:examples",
+    "test:nocompile": "npm run test:only && npm run test:runtime && npm run lint && npm run mocha:examples",
     "test:only": "nyc --reporter=html --reporter=text-summary npm run mocha:test",
+    "test:runtime": "TZ=America/Los_Angeles wdio wdio.conf.js",
+    "test:runtime:generate": "rm -Rf test-runtime/resources && VL_GENERATE_TESTS=true npm run test:runtime",
```

Comment:
The reason I chose this name is because generation of the SVG snapshots is inherently tied to the test cases (i.e., it is not a pure "build" step). One could imagine structuring the code to be the complement (i.e., only run assertions if a `TEST` environment variable is present), but that seems less clean than the current approach. 

I will update the two Markdown guides.